### PR TITLE
Add documentation for noaaweather sensor integration

### PR DESCRIPTION
**Pull request by rewsssiestedbo**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23875

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/noaaweather.markdown [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## {% linkable_title Configuration %}
+
+To add NOAA/NWS Weather to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: noaaweather
+    useragent:	email@example.com
```
Comment (klaasnicolaas) [inline source/_components/noaaweather.markdown:27]: I think that we can remove the `useragent`, only required items in the first example.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/noaaweather.markdown [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: "NOAA/NWS Sensor"
+description: "How to integrate NOAA/National Weather Service observations within Home Assistant."
+date: 2019-04-15 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo:
```
Comment (klaasnicolaas) [inline source/_components/noaaweather.markdown]: I have added already the variable for you, but could you add a logo of the integration?

Comment (MatthewFlamm) [reply source/_components/noaaweather.markdown]: This integration will use the same logo(s) as the one in [#9372](https://github.com/home-assistant/home-assistant.io/pull/9372).  The same comment will apply that we need to request permission.  We are not using unmodified data products, both this platform/integration and the one in #9372 use minimally altered, but not unaltered data.

**Comment by MatthewFlamm** [comment]
Similar to comment in the backend PR, this one should be merged with [#9372](https://github.com/home-assistant/home-assistant.io/pull/9372) as they are the same data source, but one for sensor and one for weather platform.

File: source/_components/noaaweather.markdown (hunk 2 above)
Comment (MatthewFlamm) [reply source/_components/noaaweather.markdown]: I retract this, see comment in #9372

File: source/_components/noaaweather.markdown [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+description: "How to integrate NOAA/National Weather Service observations within Home Assistant."
+date: 2019-04-15 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: noaa.png
+ha_category:
+  - Sensor
+ha_release: 0.94
```
Comment (klaasnicolaas) [inline source/_components/noaaweather.markdown:13]:
```suggestion
ha_release: 0.96
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Pull request closed without merging**